### DISCUSSION TITLE
Share payouts with dependencies

### DISCUSSION THREAD

**Discussion by LostLuma** [modrinth/.github#118] [Feature requests]
Mod authors should have the ability to give part of their revenue to projects they depend on, similarly to the way modpacks function right now.

Often significant "invisible" contributions to a project come from libraries and the library authors should be able to be rewarded for putting effort into creating and maintaining their work, since when a library is directly included within a mod the end user will not visit the library page and generate ad revenue for it.

**Comment by TechPro424** [comment]
It should be made sure that this system is only for embedded dependencies
Because users will have to download required/optional dependencies manually anyway

**Reply by triphora** [reply]
It should probably be allowed for required and optional dependencies too, because oftentimes those dependencies are downloaded through the API which do not contribute to payouts

**Reply by ghost** [reply]
When you download dependencies through the API (maybe through a launcher or the like), it's likely that you download the dependant the same way, so there isn't any money coming in in the first place.

**Reply by TechPro424** [reply]
I agree with hyper
